api/pom.xml

```diff
@@ -91,6 +91,10 @@
          <groupId>org.springframework</groupId>
          <artifactId>spring-context-support</artifactId>
       </dependency>
+      <dependency>
+				<groupId>org.springframework</groupId>
```

Comment:
This dependency is triggering a security alert. What do you think about using html-encoding functions from OWASP? These are already being used in the web section of openmrs-core: https://github.com/openmrs/openmrs-core/blob/722385734baabc0ee2e5a0d05fb1680d4ead8700/web/pom.xml#L127